docs/src/pages/component-demos/lists/CheckboxList.js

```diff
@@ -40,10 +40,12 @@ class CheckboxList extends Component {
   render() {
     const classes = this.props.classes;
 
+    /* eslint-disable prefer-spread, react/no-array-index-key */
     return (
       <div className={classes.root}>
         <List>
-          {Array.from({ length: 4 }, (v, k) => k).map(index =>
+          {/* $FlowFixMe */}
```

Comment:
More context for all of the added `$FlowFixMe`'s:

https://github.com/facebook/flow/issues/1561